INSTALL.md

```diff
@@ -0,0 +1,128 @@
+# Development on documentation
+
+## tl;dr
+
+**You will need to use following `make targets` before creating a pull request!**
+
+1. `make install`
+2. `make check`
+
+## Features
+
+* Linting of markdown documents
+* Spell checking
+* Link checking
+
+## Specifications
+
+This repository checks against following specification:
+
+* [Markdown Commonmark](https://spec.commonmark.org/)
+
+### Languages
+
+Supported languages are:
```

Comment:
Which English flavor/flavour do we support? US or UK?

Reply:
so far, most documents are in us ("-ze")